index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+    <div class="nav-container">
+
+        <nav class="nav-bar">
+            <div class="nav-toggle"><img src="images/Enabled.png"></div>
+            <div class="nav-item" id="box1"><a href="#">Portfolio</a></div>
+            <div class="nav-item" id="box2"><a href="#">About</a></div>
+            <div class="nav-item" id="box3" ><a href="#"> Contact</a></div>
+        </nav>
+    </div>
+    </header>
+    <div class="main-container">
+
+        <div id="main-item1">
+            <h2>Hey there. <span id="main-span1">I'm Kenny</span>. <span id="main-span2">I'm a software developer</span></h2>
+        </div>
+        <div id="main-item2">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+        </div>
+        <div id="main-item3">
+            <img src="images/Social-media.png">
+        </div>
```

Comment:
social media icons are used as links to redirect the user to your accounts kindly use each of them as icon wraped in an anchor a tag with a valid href link to your account and since you have a list of them please make sure to put them in ul tag 
exp `<ul><li><a href="" ><img .../></a></li></ul>`